llpc/builder/llpcBuilderImplImage.cpp

```diff
@@ -907,11 +907,21 @@ Value* BuilderImplImage::CreateImageLoad(
     pCoord = HandleFragCoordViewIndex(pCoord, flags, dim);
 
     uint32_t dmask = 1;
-    Type* pTexelTy = pResultTy;
+    Type* pOrigTexelTy = pResultTy;
     if (auto pStructResultTy = dyn_cast<StructType>(pResultTy))
     {
-        pTexelTy = pStructResultTy->getElementType(0);
+        pOrigTexelTy = pStructResultTy->getElementType(0);
+    }
+
+    Type* pTexelTy = pOrigTexelTy;
+
+    bool is64Bit = (pOrigTexelTy->getScalarSizeInBits() == 64);
```

Comment:
This doesn't seem to be reassigned anywhere; can you add `const` to make it more obvious?

Reply:
Sure, will call it isInt64Ty to better match the handling.